config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml

```diff
@@ -47,3 +47,38 @@ periodics:
       resources:
         requests:
           memory: "8Gi"
+
+- name: ci-kubernetes-e2e-windows-gce
+  decorate: true
+  extra_refs:
+  - org: yujuhong
+    repo: gce-k8s-windows-testing
+    base_ref: master
+    path_alias: k8s.io/gce-k8s-windows-testing
+  interval: 2h
+  labels:
+    preset-k8s-ssh: "true"
+    preset-service-account: "true"
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/latest
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=8
+      - --provider=gce
+      - --gcp-nodes=2
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/gce-k8s-windows-testing/run-e2e.sh
+      - --test-cmd-args=--ginkgo.focus=\[Conformance\] --minStartupPods=8
+      - --timeout=720m
```

Comment:
(note that our default pod timeout is 60m, this 720m is a no-op and probably too long?)

Reply:
You're right. I set to to a long timeout while still experimenting & testing. I just checked the existing runs, and 180m should be more than enough.

Reply:
Note that we also have entrypoint level timeout like https://github.com/kubernetes/test-infra/blob/d36768faca2dc96d9ec16069d07527d4629f64e9/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml#L15-L16, which actually defaults to https://github.com/kubernetes/test-infra/blob/91186ea67814300ed42df1188bdd2357d1d86bb2/prow/entrypoint/run.go#L58 - so if kubetest takes...say... ~150m to run, it will abort at 120m by default afaik... just FYI :-)

Reply:
Ah....good to know. Thanks. I changed the timeouts to 120m so they match.